index.html

```diff
@@ -1,24 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,300,600,800' 
+		rel='stylesheet' type='text/css'>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
```

Comment:
Wasn't part of the assignment, but it's generally a good idea to get in the habit of updating these meta tags.